drivers/SmartThings/harman-luxury/src/api/apis.lua

```diff
@@ -29,27 +20,6 @@ local APIs = {}
 
 --- system APIs ------------------------------------------
 
---- get UUID from Harman Luxury on ip
----@param ip string
----@return string|nil, nil|string
-function APIs.GetUUID(ip)
-  return get.String(ip, UUID_PATH)
-end
-
---- get MAC address from Harman Luxury on ip
----@param ip string
----@return string|nil, nil|string
-function APIs.GetMAC(ip)
-  return get.String(ip, MAC_PATH)
-end
-
---- get Member ID from Harman Luxury on ip
----@param ip string
----@return string|nil, nil|string
-function APIs.GetMemberId(ip)
-  return get.String(ip, MEMBER_ID_PATH)
-end
-
 --- get device manufacturer name from Harman Luxury on ip
 ---@param ip string
 ---@return string|nil, nil|string
```

Comment:
Are these REST calls still here because they're needed before the REST connection is open, or because there's no equivalent WebSocket message?